src/main/resources/openapi/courthouses.yaml

```diff
@@ -0,0 +1,106 @@
+openapi: 3.0.1
+info:
+  version: 0.0.1
+  title: Modernised DARTS
+  description: |-
+    Modernised DARTS (Digital Audio Recording and Transcription Service).
+servers:
+  - url: http://localhost:4550/
+paths:
+  /courthouses:
+    post:
+      tags:
+        - Courthouses
+      summary: Adds a courthouse record with supplied details.
+      requestBody:
+        $ref: '#/components/requestBodies/CourtHousePost'
+      responses:
+        '201':
+          description: Created
+        '500':
+          description: Internal Server Error
```

Comment:
should we have some sort of "already exists" error?

Reply:
ACCEPT. added a 409 spec.